SEED-platform/seed
# Multiple cycle upload

#### Any background context you want to provide?

#### What's this PR do?
Allows you to upload a spreadsheet with data across multiple cycles.

#### How should this be manually tested?
Upload the file '[Sample_MultipleCycles.xlsx](https://github.com/SEED-platform/seed/blob/multiple_cycle_upload/seed/tests/data/Sample_MultipleCycles.xlsx)' and select the 'Multiple Cycle Upload' checkbox.

![MultipleCycle Checkbox](https://github.com/SEED-platform/seed/assets/6326294/09ed99f4-261e-403c-9c92-59abff278c07)

Additionally, ensure that the SEED column 'year_ending' is mapped to the "Year Ending" column from the 'Date File Header'.

#### What are the relevant tickets?
#3960 #4053 #3181 

#### Screenshots (if appropriate)
